utils/releaser/src/utils.rs

```diff
@@ -69,3 +70,72 @@ pub fn root() -> PathBuf {
     }
     project_root
 }
+
+pub fn determine_new_version(bump_type: Option<String>) -> Option<String> {
+    let mut current_version: Vec<i32> = core_version()
+        .split('.')
+        .map(|f| f.parse().unwrap())
+        .collect();
+
+    let bump_type = bump_type.unwrap_or_else(|| "patch".to_string());
+    match bump_type.as_ref() {
+        "major" => current_version[0] += 1,
+        "minor" => current_version[1] += 1,
```

Comment:
if a minor version is bumped, the patch version should be set to 0, similar deal w major